source/doc-pages/introduction.md

```diff
@@ -1,35 +1,62 @@
-Ruby Object Mapper (ROM) is a Ruby library with the goal to provide powerful object
-mapping capabilities without limiting the full power of your datastore. ROM is based
-on several concepts and decisions that differentiate it from your normal Ruby ORM:
+Ruby Object Mapper (ROM) is a Ruby persistence library with the goal to provide 
+powerful object mapping capabilities without limiting the full power of your 
+datastore. 
 
-* Provide minimum infrastructure to handle mapping and persistence
-* Provide higher-level abstractions on top of lower-level components
-* Querying the datastore is considered a private implementation detail
-* Abstract query interfaces are sources of unnecessary complexity
-* Reading and mutating data are distinct concerns
-* Simple to use the underlying datastore when desired
-* Be kind to the runtime environment
+More specifically, ROM exists to:
 
-These [core concepts](/introduction/overview) are implemented in ROM with
-[**adapters**](/introduction/adapters), [**relations**](/introduction/relations),
-[**mappers**](/introduction/mappers), and [**commands**](/introduction/commands).
+* Isolate the application from persistence details
+* Provide minimum infrastructure for mapping and persistence
+* Provide shared abstractions for lower-level components
+* Provide simple use of the underlying datastore when desired
 
-On top of those components ROM provides **[Repository](/guides/basics/repositories/)**
-interface for convenient access to your data.
+These [core features](/introduction/overview) are implemented in ROM through 
+[**Repository**](/guides/basics/repositories), which in turn uses the fundamental 
+building blocks [**adapters**](/introduction/adapters), [**relations**](/introduction/relations),
+[**mappers**](/introduction/mappers), and [**commands**](/introduction/commands). 
```

Comment:
This is where graphic design and typographic hierarchy *really helps*.

Maybe if we pulled the `### Get Started!` paragraph up from the bottom and put it above here, then headed this paragraph with `### Core Features`, it might answer your question @solnic?

What do you think @robinetmiller?